Formula/p/pythran.rb

```diff
@@ -84,9 +89,7 @@ def arc_distance(theta_1, phi_1, theta_2, phi_2):
         return distance_matrix
     EOS
     # Test with configured gcc to detect breakages from gcc major versions and for OpenMP support
-    with_env(CC: nil, CXX: nil) do
-      system pythran, "-DUSE_XSIMD", "-fopenmp", "-march=native", testpath/"arc_distance.py"
-    end
```

Comment:
We shouldn't change this.

Reply:
I see 👍

Reply:
now seeing the same in https://github.com/Homebrew/homebrew-core/pull/196198 maybe something not from pythran?